src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.util.List;
+
+/**
+ * TaskList class that is used by duke class to deal with the list of task.
+ * It allows adding and deleting of tasks from the list.
+ */
+public class TaskList {
+    private List<Task> taskList;
+
+    public TaskList(List<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Add tasks to the list.
```

Comment:
Should it be "Adds" instead? Same for the other javadoc comments :)
```suggestion
     * Adds tasks to the list.
```